queries/private-cloud/helpers.ts

```diff
@@ -7,17 +7,19 @@ export async function getPrivateCloudProjectsQuery({
   cluster,
   userEmail,
   ministryRoles,
+  active,
 }: {
   searchTerm?: string | null;
   ministry?: string | null;
   cluster?: string | string[] | null;
   userEmail?: string | null;
   ministryRoles?: string[];
+  active?: string | null;
 }) {
+  const setActive = active === 'false' ? false : true;
```

Comment:
`setActive` sounds more like a `method` rather than a variable.
something like `isActive` might be better name? 🤔